pkgs/by-name/cn/cntb/package.nix

```diff
@@ -0,0 +1,30 @@
+{ buildGoModule
+, lib
+, fetchFromGitHub
+, stdenv
+, ...
+}: buildGoModule rec {
+  pname = "cntb";
+  version = "1.4.6";
+
+  src = fetchFromGitHub {
+    owner = "contabo";
+    repo = "cntb";
+    rev = "v${version}";
+    hash =
+      if stdenv.isDarwin
+      then "sha256-cUEzX10E1fBC66sAYoATY1q9+I/U+t0Tyi3HfmwQzJ4="
+      else "sha256-MB5rxZCQtq2xaiTC6dGDwdt9VKxpWwBlI9EI91fes/4=";
```

Comment:
It's possible to use case-sensitive filesystems on Darwin, and case-insensitive filesystems on Linux, so this isn't quite right.  Since the files aren't actually needed, the right thing to do would be to remove one of them in `postFetch`, which is run before the hash is generated.  (Or even better, point this out to upstream and ask "wtf?".)

Reply:
I just created an issue upstream: https://github.com/contabo/cntb/issues/34

> Since the files aren't actually needed, the right thing to do would be to remove one of them in postFetch

I ended up removing the entire `docs` folder, it seems to work, let's see if ofborg correctly builds it on darwin using the same hash.

Another advantage of this approach is that @r-ryantm should be able to send PRs, I don't think it would have been with `if` expression.